Source/UI/NewUI/CommonUI/View/TableViewCell/CKOCellTextField.swift

```diff
@@ -1,65 +1,51 @@
 import UIKit
-import PhoneNumberKit
 
-protocol BillingFormTextFieldCellDelegate: AnyObject {
+protocol CKOCellTextFieldDelegate: AnyObject {
     func updateCountryCode(code: Int)
     func textFieldShouldBeginEditing(textField: UITextField)
     func textFieldShouldReturn()
     func textFieldShouldEndEditing(textField: UITextField, replacementString: String)
     func textFieldShouldChangeCharactersIn(textField: UITextField, replacementString string: String)
 }
 
-final class CKOTextFieldCell: UITableViewCell {
-    weak var delegate: BillingFormTextFieldCellDelegate?
+final class CKOCellTextField: UITableViewCell {
```

Comment:
The name should reflect that this is a cell, otherwise that is completely missed for a long time.

Reply:
please add suggestion here using <Cmd+g>

Reply:
```suggestion
final class CKOTextFieldCell: UITableViewCell {
```

Thanks for the shortcut 👍. I think the previous name was clearer as it ends in _cell_, with the description prior, and this is most importantly a tableViewCell as shown by the inheritance from `UITableViewCell`.

Reply:
Updated